tests/EmailValidator/Validation/DNSCheckValidationTest.php

```diff
@@ -14,50 +16,94 @@ public function validEmailsProvider()
     {
         return [
             // dot-atom
-            ['Abc@example.com'],
-            ['ABC@EXAMPLE.COM'],
-            ['Abc.123@example.com'],
-            ['user+mailbox/department=shipping@example.com'],
-            ['!#$%&\'*+-/=?^_`.{|}~@example.com'],
+            ['Abc@github.com'],
```

Comment:
`example` is not a valid domain?

Reply:
It is a valid domain, but it also has a "Null MX" record, so now fails!

Reply:
However I'd like to find non-commercial examples. A quick search has failed me. I thought `mail.example.com` would work but it doesnt. Do you know any?

Reply:
ietf.org, maybe?

Reply:
That would be perfect!